### PR TITLE
Pcap selection page

### LINKED ISSUE #42: PCAP file loading page
Create an webpage that allows the user to select a PCAP files to use for our visualizations.

### PR DESCRIPTION
**Pull request by Erik-Handeland**
# Description

Add a summary of the change and which issue is fixed, including motivation, context, dependencies, etc. 

# Checklist:

- [ ] Code follows the style guidelines
- [ ] Code commented in relevant places
- [ ] Changes to necessary documentation made
- [ ] Has test cases for code.

# Tests Performed:

- [ ] Upload data to Datastore
- [ ] Retreive data from Datastore
- [ ] Map Visualization 
- [ ] Network Map Visualization 
- [ ] List other manual test cases performed

### CODE REVIEW

**Comment by Erik-Handeland** [comment]
Screen Shot 2020-08-17 at 4 50 58 PM

Clicking the row sends a form to servlet which stores entity name as a session object.

File: src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java [hunk 1]
```diff
@@ -19,6 +25,35 @@ public UtilityPCAPDaoImpl() {
...
+      catch(Exception e)
+       {
+          return null;
+        }
+  }
+
+  public void setSessionEntity(HttpServletRequest request, String text) {
+    HttpSession session = request.getSession();  
+    session.setAttribute("entityName",text);  
+  }
```
Comment (arunk054) [inline src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java]: These methods dont belong in a Utility Class, it is a separate class for session Management, like `class SessionManager`

Comment (arunk054) [reply src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java]: nevermind keep it here for now but make sure to keep all methods static.

**Review by arunk054 — APPROVED** [review]

File: src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java (hunk 1 above)
Comment (Erik-Handeland) [reply src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java]: ran into issues declaring static methods while having an interface, ended up created the SessionManager class

**Pull request merged**